test/model/content_test.dart

```diff
@@ -1503,16 +1534,20 @@ void main() {
   testParseExample(ContentExample.tableMissingOneBodyColumnInMarkdown);
   testParseExample(ContentExample.tableWithDifferentTextAlignmentInColumns);
   testParseExample(ContentExample.tableWithLinkCenterAligned);
+  testParseExample(ContentExample.orderedListLargeStart);
+  testParseExample(ContentExample.orderedListCustomStart);
 
   testParse('parse nested lists, quotes, headings, code blocks',
     // "1. > ###### two\n   > * three\n\n      four"
     '<ol>\n<li>\n<blockquote>\n<h6>two</h6>\n<ul>\n<li>three</li>\n'
         '</ul>\n</blockquote>\n<div class="codehilite"><pre><span></span>'
         '<code>four\n</code></pre></div>\n\n</li>\n</ol>', const [
-      ListNode(ListStyle.ordered, [[
+      OrderedListNode(
+        start: 1,
+        items: [[
         QuotationNode([
```

Comment:
nit: keep the existing formatting:
```suggestion
      OrderedListNode(start: 1, items: [[
        QuotationNode([
```

In particular the version in the current revision of the PR doesn't have correct indentation: the QuotationNode inside `items` needs to be indented more deeply than the line `items` starts on.